<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stream Snatcher</string>
  </property>
  <widget class="QGroupBox" name="groupBox_config">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QLabel" name="label_config_url">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_config_refererUrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Referer URL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_config_segment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Segment</string>
    </property>
    <widget class="QLabel" name="label_config_segment_videoLength">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Video Length (hh:mm:ss)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_config_segment_manualInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual Input</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_config_segment_autoCalculation">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Calculation</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_config_segment">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_config_segment_videoLength">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>01:23:45</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_config_segment_timePerSegment">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Per Segment</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_config_segment_timePerSegment">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_config_segment_apply">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_config_refererUrl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>https://example.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_config_url">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>741</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>https://example.com/video(*).ts</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_config_output">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_config_output_folderPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_config_output_folderPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_config_output_fileDialog">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_config_output_fileName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_config_output_fileName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>output_file</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_config_zeroPadding">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zero-Padding</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_config_zeroPadding">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>901</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Log</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser_log">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>21</y>
      <width>871</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_download">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>550</y>
     <width>891</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
